consult-dir.el

```diff
@@ -93,12 +97,29 @@
 
 (defcustom consult-dir-shadow-filenames t
   "Shadow file names instead of replacing them when using `consult-dir'."
+  :group 'consult-dir
+  :type 'boolean)
+
+(defcustom consult-dir-include-tramp-hosts t
```

Comment:
I would prefer not to add this flag. Instead, remove the `:enabled` clause of the corresponding sources and don't include the docker sources in `consult-dir-sources` by default.

Reply:
OK, I'll make it clear in the documentation that these sources exist :)